packages/xo-web/src/common/intl/messages.js

```diff
@@ -786,6 +786,10 @@ const messages = {
   // ----- host stat tab -----
   statLoad: 'Load average',
   // ----- host advanced tab -----
+  editHostIscsiIqnTitle: 'Edit iSCSI IQN',
+  editHostIscsiIqnMessage: 'Are you sure you want to edit the iSCSI IQN?',
+  editHostIscsiIqnInfo:
+    'It must be unique in order to uniquely identify the host on the network.',
```

Comment:
There have two warnings to notify. The iSCSI IQN must be unique and the host will loose the connection with the attached ISCSI SRs.